contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -166,6 +166,7 @@ script: |
     make ${MAKEOPTS} -C src check-symbols
     make install DESTDIR=${INSTALLPATH}
     if [ -f src/electrscash ]; then
+        strip src/electrscash
```

Comment:
we should save a copy before stripping so that we can debug.  We keep this file internally rather than releasing it.  Se what I do for bitcoind starting at line 172 (just below this comment).
Maybe:
cp -f src/electrscash ${OUTDIR}/electrscash-${i}.debug

Reply:
makes sense, will do